packages/components/src/styles/tile/button-link-tile-styles.ts

```diff
@@ -39,6 +39,8 @@ export const getButtonLinkTileStyles = (
         margin: 0,
         ...textLargeStyle,
         hyphens: 'inherit',
+        overflowWrap: 'anywhere',
```

Comment:
maybe add a comment to both css props why you have chosen them.

Reply:
added comment